reference/authorisation_model.rst

```diff
@@ -0,0 +1,110 @@
+
+The model
+---------
+
+JAAS authorisation model reshapes the `Juju permission model <https://juju.is/docs/juju/user-permissions>`_ into a ReBAC paradigm. The OpenFGA authorisation model used by JAAS is defined as:
+
+.. code:: text
+
+    model
+        schema 1.1
+
+    type applicationoffer
+    relations
+        define administrator: [user, user:*, group#member] or administrator from model
+        define consumer: [user, user:*, group#member] or administrator
+        define model: [model]
+        define reader: [user, user:*, group#member] or consumer
+
+
+    type cloud
+    relations
+        define administrator: [user, user:*, group#member] or administrator from controller
+        define can_addmodel: [user, user:*, group#member] or administrator
+        define controller: [controller]
+
+    type controller
+    relations
+        define administrator: [user, user:*, group#member] or administrator from controller
+        define audit_log_viewer: [user, user:*, group#member] or administrator
+        define controller: [controller]
+
+    type group
+    relations
+        define member: [user, user:*, group#member]
+
+    type model
+    relations
+        define administrator: [user, user:*, group#member] or administrator from controller
+        define controller: [controller]
+        define reader: [user, user:*, group#member] or writer
+        define writer: [user, user:*, group#member] or administrator
+
+    type serviceaccount
+    relations
+        define administrator: [user, user:*, group#member]
+
+    type user
+
+Here is the directed graph illustration of the above model. In this figure, purple and green nodes represent entity types and relations, respectively. The dashed lines show the internal indirect relationships among relations defined on the entity type.
+
+.. image:: images/authorisation_model.png
+
+
+Tuples
+------
+
+In ReBAC, a *tuple* is a plain data structure that represents the relationship between two entities. So, a tuple has three components:
+
+1. Entity A (or *Object*): the entity that receives (or uses) the relation.
+2. Relation: the type of relationship.
+3. Entity B (or *Target*): the entity that provides the relation.
+
+.. note::
+    JAAS terminology is slightly different from OpenFGA. In OpenFGA, *Entity A* and *Entity B* are called *User* and *Object*, but in JAAS, they are called *Object* and *Target (object)*. The reason behind this is to prevent confusion over real users and relationship users.
+
+For example, if a ``user`` named ``alice@canonical.com`` has the ``member`` relationship with a group named ``foo``, then the tuple that represents this relation will look like this:
+
+.. code:: yaml
+
+    object:   user:alice@canonical.com
+    relation: member
+    target:   group:foo
+
+This reads as: "an entity of type ``user``, named ``alice@canonical.com``, has ``member`` relationship to an entity of type ``group``, named ``foo``.
+
+
+Manipulating tuples
```

Comment:
This feels like it belongs in a separate reference. Or a how-to.

Reply:
Just separated data and model into two different topics.